ui/pages/bridge/bridge.util.ts

```diff
@@ -124,7 +125,12 @@ export async function fetchBridgeTokens(
 
   tokens.forEach((token: unknown) => {
     if (
-      validateResponse<SwapsTokenObject>(TOKEN_VALIDATORS, token, url) &&
+      validateResponse<SwapsTokenObject>(
+        TOKEN_VALIDATORS.concat(TOKEN_AGGREGATOR_VALIDATORS),
```

Comment:
This is called in the background. I wonder if this is causing the increase in memory usage